modules/ROOT/pages/security/single-sign-on.adoc

```diff
@@ -47,38 +58,122 @@ Accessing Aura with SSO requires:
 
 . The _Project ID_ of the projects you want to use SSO for. Click on the project settings to copy the ID.
 
 . The name of your IdP
+
+== Azure configuration
+
+=== Azure SSO configuration
+
+. In the *Azure Portal*, go to *App Registrations* and then *New Registration*.
+
+. Add a name for the new app registration and select *Register*. 
+Skip redirect URI’s for now.
+
+. On the app overview page, take note of the Application (client) ID.
+
+. Select the *Client Credentials* link to open the client credentials page.
+
+. Create a new secret and *copy the Value field*, it won’t be visible after leaving the page.
+
+. Go back to the *App Overview* page and open the *App Endpoints* and take note of the OpenID Connection metadata document URI
+
+. Under *Authentication* in the left-hand navigation, setup redirect URLs:
+
+.. Adding a new Web platform 
+.. Enter `https://login.neo4j.com/login/callback` as the redirect URI.
+
+=== Create an Azure SSO configuration in the Aura console
+
+. Go to *Organization Settings*
+
+. Select if you want the SSO configuration to be applied to organization logins, to specific projects within the organization, or both.
+
+. For IdP Type select *Microsoft Entra ID*
+
+. For Client ID enter the *Application (client) ID* from the Azure app
+
+. For Client Secret enter the client secret value (not secret id) from the secret you created in the Azure app
+
+. For Discovery URI enter the *OpenID Connect metadata document URI*
+
+. Configure any additional settings as needed
+
+. Select *Create*
+
+=== Token request scopes
+
+When requesting the token from Azure, the scopes Aura sends are:
+
+* `openid` access to a unique identifier to identify the user. 
+
+* `profile` access to basic profile information.
+
+* `email` contains the user's email address.
+
+This will result in Azure asking for consent to display details related to these scopes. 
+For more information, see link:https://auth0.com/docs/get-started/apis/scopes/openid-connect-scopes#standard-claims[OpenID Connect Scopes]
+
+== Pilot testing
+
+*Organization-level testing*
+
+Keep the Email/password or Google log-in method enabled, so that if SSO fails, you can still access the Aura console and adjust the configuration.
+
+Login to the Aura console with SSO.
+
+*Project-level testing*
+
+Keep the User/password login enabled, so that if SSO fails, you can still access the database.
+Create a new instance in the project with SSO and login.
+
+== FAQ
+
+*Can users get roles added to them in Aura console via SSO and a group to role mapping?*
+
+No, users must be granted the role on the org via Aura console invites and access management like with any other organization.
+
+*Why am I unable to connect to the instance after completing the SSO login, the connection is showing as unconnected?*
+
+Ensure that the email field is provided on your user in Microsoft Entra ID. 
+If it already is, contact support for further assistance.
```

Comment:
I'd recommend to rewrite this section to general recommendations. I'm not a fan of FAQs, we can just write out the information without making it into a question. If it makes sense.